### DISCUSSION TITLE
Dropdown lazyload with API call

### DISCUSSION THREAD

**Discussion by vigneshnb** [primefaces/community#2241] [PrimeNG]
How can I use lazy load with api call? I see in demo, it's predefined list of items only. then update value based on index when scroll (first, last) property.  

my question is,

1. how to set last property value as dynamically? my api return total_count property.
2. shall I use page and limit parameters to call the api?

**Comment by Logic-Bits** [comment]
https://github.com/primefaces/primeng/issues/5955
this is exactly this - it got closed
